superset-frontend/src/chart/Chart.jsx

```diff
@@ -74,6 +74,7 @@ const defaultProps = {
   setControlValue() {},
   triggerRender: false,
   dashboardId: null,
+  chartStackTrace: null,
```

Comment:
Is this change intended? I didn't see any reference within this PR.

Reply:
Yes. I was seeing `chartStackTrace` being `undefined` on the initial rendering, then immediately updated to `null`. Thought it makes sense to have a consistent value.